OloEngine/src/OloEngine/Scripting/Lua/LuaScriptGlue.cpp

```
@@ -556,6 +1323,26 @@ namespace OloEngine
             gpAxisTable["RightTrigger"] = static_cast<u8>(GamepadAxis::RightTrigger);
         }
 
+        // --- KeyCode constants (auto-generated from OLO_KEY_LIST in KeyCodes.h) ---
+        {
+            auto keyTable = lua.create_named_table("KeyCode");
+            // clang-format off
+#define OLO_BIND_KEY(name, val) keyTable[#name] = static_cast<KeyCode>(val);
```

Clang (14.0.6)

[warning] function-like macro 'OLO_BIND_KEY' used; consider a 'constexpr' template function

(cppcoreguidelines-macro-usage)